adserver/admin.py

```diff
@@ -210,6 +213,7 @@ class AdvertisementsInline(AdvertisementMixin, admin.TabularInline):
     model = Advertisement
 
     can_delete = False
+    list_per_page = 50  # make page load a bit faster
```

Comment:
I don't think there will be a ton of 50+ ad flights. I also don't think this actually has any effect. It [isn't one of the fields shared with ModelAdmins](https://docs.djangoproject.com/en/2.2/ref/contrib/admin/#inlinemodeladmin-options).